### PR TITLE
Extend Scylla Manager E2E test with repair task update and deletion

### PR DESCRIPTION
**Pull request by rzetelskik**
**Description of your changes:** This PR extends the existing manager integration E2E test with update and deletion of the repair task to increase coverage of tested operations.

**Which issue is resolved by this Pull Request:**
Resolves #

/kind feature
/priority important-longterm
/cc

### CODE REVIEW

**Comment by rzetelskik** [comment]
both flakes seem related to PDs
/retest

**Comment by rzetelskik** [comment]
seems related to PDs again
/retest

**Comment by rzetelskik** [comment]
timeouts again
/test images
/retest

File: test/e2e/set/scyllacluster/scyllamanager.go [hunk 1]
```diff
@@ -132,6 +132,49 @@ var _ = g.Describe("Scylla Manager integration", func() {
...
+		framework.By("Updating the repair task for ScyllaCluster")
+		sc, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Patch(
+			ctx,
+			sc.Name,
+			types.JSONPatchType,
+			[]byte(`[{"op":"replace","path":"/spec/repairs/0/parallel","value":1}]`),
+			metav1.PatchOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(sc.Spec.Repairs[0].Parallel).To(o.Equal(int64(1)))
```
Comment (tnozicka) [inline test/e2e/set/scyllacluster/scyllamanager.go:144]: nit: please assert the opposite before the patch as well

Comment (tnozicka) [reply test/e2e/set/scyllacluster/scyllamanager.go:144]: i suppose status assert would be useful too

**Review by tnozicka — COMMENTED** [review]
/approve

**Review by zimnx — APPROVED** [review]
/assign @tnozicka

File: test/e2e/set/scyllacluster/scyllamanager.go (hunk 1 above)
Comment (rzetelskik) [reply test/e2e/set/scyllacluster/scyllamanager.go:151]: added, thanks

**Review by tnozicka — APPROVED** [review]
lgtm
/assign @zimnx

**Pull request merged**